.github/ISSUE_TEMPLATE/reference-architecture.md

```diff
@@ -0,0 +1,118 @@
+name: "Reference Architecture Submission"
+description: "Submit a Reference Architecture for Community Review"
+title: "[Reference: ]"
+labels: ["reference-architecture"]
+body:
+ - type: markdown
+   attributes:
+     value: |
+       **Purpose of This Issue**
+ 
+       This is a submission to the reference architecture process.
+
+       Add more information here for submitters...
+ 
+ - type: input
+   id: name
+   attributes: 
+     label: Name
+     description: What is the name of this architecture submission?
+     placeholder: e.g., My Reference Architecture
```

Comment:
I think we need to give more guidance here. We can hint at having org name+tech area, or domain name+tech area, or?